core/src/core/registerDefaultDependencies.tsx

```diff
@@ -20,19 +20,19 @@ function registerDefaultDependencies(container: IDiContainer, name: string) {
   container.bind(ComponentLoader).toConstantValue(new ComponentLoader());
 
   /** router */
-  container.bind<IRouteConfig[]>(serviceIds.routes).toConstantValue([
+  container.bind<RouteConfigType>(serviceIds.routes).toConstantValue([
     {
-      /* eslint-disable react/display-name */
-      component: () => <h1>Hello World</h1>,
-      /* eslint-enable react/display-name */
       name: 'home',
-      pattern: '/',
-    },
+      path: '',
+      action: () => {
+        return () => <h1>Hello World</h1>
+      },
+    }
   ]);
 
-  container.bind(serviceIds.routerUrlFormatStrategy).toDynamicValue(() => {
-    return new DefaultRouterStrategy(null, null);
-  });
+  container.bind(serviceIds.routerSerializeRouteInUrlStrategy).toConstantValue(
+    new Path()
```

Comment:
PathSerializer ist ja der "Standard" jetzt. Hast du mal getestet, wie es ist, wenn man in irgendeiner route ist und dann mit F5 neu lädt?
Soweit ich weiß funktioniert das mit SPAs nicht, da wir folderstruktur nicht abbilden sondern das alles code seitig gerouted werden muss.